go/ct/st/state.go

```diff
@@ -256,9 +256,16 @@ func (s *State) Eq(other *State) bool {
 			s.ReturnData == other.ReturnData
 	}
 
+	// All PC beyond the end of the code are equal, as all PCs in this range
+	// point to the STOP instruction.
+	equivalentPc := s.Pc == other.Pc
```

Comment:
nit: because type is bool, a boolean expression as initializer would read better than control flow.

Reply:
Done